src/components/form/range/dual_range.js

```diff
@@ -404,30 +404,23 @@ export class EuiDualRange extends Component {
     );
 
     const thePopover = showInputOnly ? (
-      <EuiPopover
-        // ownFocus={popoverShouldOwnFocus}
-        button={
+      <EuiInputPopover
+        input={
           <EuiFormControlLayoutDelimited
             fullWidth={fullWidth}
             compressed={compressed}
             startControl={minInput}
             endControl={maxInput}
           />
         }
-        isOpen={this.state.isPopoverOpen}
-        closePopover={this.closePopover}
-        // zIndex={popoverZIndex}
         className={classNames('euiRange__popover', {
           'euiRange__popover--fullWidth': fullWidth,
```

Comment:
This is something we do a lot with inputs, make them `fullWidth`. Perhaps we can just have `EuiInputPopover` accept a `fullWidth` prop to take care of that styling innately?

Reply:
Let me see

Reply:
So I did this, but it brings up another point: the full width popover looks weird with a full width input

<img width="872" alt="Screen Shot 2019-08-01 at 3 15 46 PM" src="https://user-images.githubusercontent.com/2728212/62324575-79d6ca80-b46f-11e9-92e2-d1f0b88a9077.png">

Reply:
Yes, and no, really it would end up being up to the maker of the input, but a lot of times we see fullWidth being applied because the area they're putting the control in is adjustable so they want the control to increase with the user's adjusted width. The creator of the component though could put a max-width on it that's larger or smaller than the default form max width.

Reply:
Works for me. I think adding the `fullWidth` prop to this new component makes it more focused on form controls, which increases the argument for a new component (thread below)